packages/web3/src/index.ts

```diff
@@ -323,7 +323,4 @@ along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
  */
 import { Web3 } from './web3';
 
-export * from './web3';
-export * from './types';
```

Comment:
We can't skip exporting types from this package as well the named export.

Reply:
why can't we skip exporting types? using muhammads branch in a test project and looks like it still gets type safety for `Web3EthInterface`

Reply:
Unfortunately we have to skip the types export due to a limitation in TS. We can't combine name and default exports in CommonJS.

https://github.com/microsoft/TypeScript/issues/2719#issuecomment-123960556